pkg/tsdb/postgres/postgres.go

```diff
@@ -42,16 +47,40 @@ func newPostgresQueryEndpoint(datasource *models.DataSource) (tsdb.TsdbQueryEndp
 	return sqleng.NewSqlQueryEndpoint(&config, &queryResultTransformer, newPostgresMacroEngine(timescaledb), logger)
 }
 
-func generateConnectionString(datasource *models.DataSource) string {
+func generateConnectionString(datasource *models.DataSource, logger log.Logger) (string, error) {
 	sslmode := datasource.JsonData.Get("sslmode").MustString("verify-full")
+
+	// Always pass SSL mode
+	sslopts := "sslmode=" + url.QueryEscape(sslmode)
+
+	// Attach root certificate if provided
+	if sslrootcert := datasource.JsonData.Get("sslRootCertFile").MustString(""); sslrootcert != "" {
```

Comment:
Isn't it written as "sslrootcertfile" in the documentation?

Reply:
y'all decided you wanted camel case in the discussion above?

Reply:
please feel free to just directly edit this branch with whatever further changes you need

Reply:
@ryankurte Ah yeah, I couldn't recall. But in that case, the documentation should be fixed for consistency.